### ISSUE TITLE
Motor Toon Grand Prix (JPN) freezes unless you yank the memory card out.

### ISSUE THREAD

**Issue by GoatWalker** [brad-lin/FreePSXBoot#87]
I don't know if this is the right place to put this but I've been having trouble with loading this particular game. So I have a legit copy and I'm using a Memcard Pro loaded with FreePSXBoot and a PSOne (the slim one). I do have the right image on my card because I was able to easily load my copy of Fire Pro Wrestling G easily with no problems.

After FreePSXBoot does it's thing, it freezes on the Sony Computer Entertainment Presents screen. I thought something was wrong, until I pulled out the memory card and it would boot normally. So as an alternative, I used another memory card I had lying around it was able to save and load fine and during all of that, I noticed that my Memcard wouldn't create another virtual memory card dedicated to the game. And I'm assuming the game is trying to read the memory card, but it's obviously "corrupt" according to the games standards.

TL;DR: Motor Toon Grand Prix (
SCPS-10001) doesn't play well with Memcard Pro installed with FreePSXBoot, it tries to read the data on the card and create a save on said card, a VMC dedicated to the game isn't being created and because of all this mess, the game freezes after the game is loaded.

**Comment by GoatWalker** [comment]
So I was doing some digging. This is more of Memcard Pro issue.
